src/Microsoft.Diagnostics.DebugServices.Implementation/LinkedListNode.cs

```diff
@@ -0,0 +1,161 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Diagnostics.DebugServices.Implementation
+{
+    /// <summary>
+    /// Simple linked list implementation. It assumes the nodes in the list inherit this class.
```

Comment:
I updated the comment to:

```
    /// <summary>
    /// Simple linked list implementation. It assumes the nodes in the list inherit this 
    /// class. The standard C# LinkedList/LinkedListNode doesn't allow it to be inherited
    /// in the (i.e. ServiceEvent) nodes making it inefficient (because LinkedListNode 
    /// contains the data node), difficult to remove/maintain the list because LinkedListNode
    /// doesn't have a remove function and LinkedList doesn't allow the nodes to be removed 
    /// during enumeration.
    /// </summary>
```

Reply:
I don't see the updated comment

Reply:
Pushed.